src/js/components/DescriptionList.js

```diff
@@ -2,16 +2,16 @@ import React from 'react';
 
 class DescriptionList extends React.Component {
   getHeadline() {
-    let {headline} = this.props;
+    let {headline, headlineClassName} = this.props;
     if (!headline) {
       return null;
     }
 
     // Wrap in headline element and classes
     return (
-      <h6 className="flush-top">
+      <h5 className={headlineClassName}>
```

Comment:
yes, the designers don't want h6 anywhere (they made this request a while ago during a design sync)